src/bundles/communication/__tests__/index.ts

```diff
@@ -0,0 +1,224 @@
+import { MultiUserController } from '../MultiUserController';
+import { GlobalStateController } from '../GlobalStateController';
+
+let multiUser = new MultiUserController();
+multiUser.setupController('broker.hivemq.com', 8884);
```

Comment:
This is a test file, so I think it's fine to hardcode.

Reply:
I think there are two distinct use-cases:

(1) playing with distributed programming in sourceacademy.org, using a public MQTT server.

(2) using our own MQTT server from sourceacademy.nus.edu.sg.

I think (1) is enough for @8kdesign 's project, so let's focus on that and get the PR merged. 

Fore (2): It would be great to re-use (1) somehow.


|